src/appleseed/renderer/kernel/rendering/tilecallbackcollection.h

```diff
@@ -0,0 +1,66 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
```

Comment:
Replace these two lines by a single one:

    Copyright (c) 2019 YOUR NAME, The appleseedhq Organization

Replace `YOUR NAME` with your name.

Same change in the .cpp file.

Reply:
Does that mean I am a part of appleseedhq? :joy:

Reply:
The comma between "YOUR NAME" and "The appleseedhq Organization" means "and" :)

But you're de facto part of "the appleseedhq contributors", yes.

Reply:
I'm honored. :)